src/views/GovernanceReviewTeam/RequestOverview.tsx

```diff
@@ -186,7 +186,7 @@ const RequestOverview = () => {
             />
             <Route
               path="/governance-review-team/:systemId/decision"
-              render={() => <Decision systemIntake={systemIntake} />}
+              render={() => <Decision systemIntake={intake} />}
```

Comment:
I've been doing something a bit different to get around rendering with values that are potentially nil. BUT I think this moves us in the right direction, and gives us two different patterns to look at.